1.20.2/23w31a.md

```diff
@@ -0,0 +1,94 @@
+
+pack | Validation for symbolic links now occurs in datapacks and resource packs.
+
+assets breaking | All textures in the `realms` namespace have been moved into the `minecraft` namespace.
+
+assets breaking | All textures containing multiple sprites in a sheet for GUI have been split into individual sprites under `textures/gui/sprites`. For example:
+* `widgets.png` has been split into `widgets/button.png` and `widgets/button_highlighted.png`.
+...
+Each sprite can now be individually overriden by a resource pack instead of replacing the entire sheet. Sprites are loaded into the `gui` atlas from the `textures/gui/sprites` directory.
+
+assets breaking | `villager2.png` has been renamed to `villager.png`.
+
+assets breaking | `icon/trial_available` and `realm_status/expires_soon` GUI sprites are animated with `.mcmeta` declarations instead of individual sprites.
+
+assets | Sprites in the gui atlas can now configure scaling behavior through a new `gui` section in `.mcmeta` files. The `gui` section in `.mcmeta` contains one `scaling` field with the following `type` options:
+* `stretch` (default) stretches the sprite across the desired space.
+* `tile` repeates the sprite across the desired space, requiring `width` and `height` fields.
+* `nine_slice` slices the sprite into 4 corners, 4 edges, and 1 center slice to be tiled across the desired space. `width`, `height`, and `border` fields are required.
+
+assets breaking | The bundle tooltip background is now drawn as a nine-sliced sprite.
+
+assets breaking | The icons for the Accessibility, Language, Realms News buttons have been split from their underlying button texture.
+
+assets breaking | The Realms Invite button is now drawn as an overlay on the normal button texture, and the highlighted texture state for this button has been removed.
+
+assets breaking | The exclamation marks on the `notification\more.png` have been removed.
+
+gamerule | Rate of accumulation of snow and ice is now affected by the `randomTickSpeed` game rule.
+
+command | Added the `/random` command for controlling random sequences, with syntax:
+* `random value|roll <range> [<sequenceId>]`: rolls a random number based on the given `range`. `roll` announces it in chat, while `value` does not
+* `random reset *|<sequenceId> [<seed>] [<includeWorldSeed>] [<includeSequenceId>]`: resets the given `sequenceId` by using a `seed` to replace it
+
+command obsolete | Added the `/execute if function` sub-command, with syntax:
+* `execute if|unless function <function> <continuation>`: executes a function/function tag and matches the return value
+
+command obsolete | Added the `/return run` sub-command, with syntax:
+* `return run <command>`: takes the return value from the `command` and returns it as the return value of the function
+
+text pack | A single backslash `\` as the last non-whitespace character of a line now allows a command to be continued on the next line
```

Comment:
`text` is only meant for text components and other client side text stuff. I would use the `command` tag for this.